.github/renovate.json5

```diff
@@ -10,63 +10,80 @@
       "enabled": false,
       "matchManagers": ["regex"],
       "matchUpdateTypes": ["major"]
-    },{
+    },
+    {
       // Disable minor version update for previous Camunda releases which will only get patch updates.
       "enabled": false,
       "matchManagers": ["regex"],
       "matchPaths": [".env"],
-      "matchBaseBranches": ["/^stable\\/8.0/", "/^stable\\/8.1/", "/^stable\\/8.2/"],
+      "matchBaseBranches": ["/^stable\\/8\\..*/"],
```

Comment:
I figured 8.3 was missing here and by that chance maybe the expression is enough and will require less maintenance?

Reply:
I think you're right. though, manually specifying the allowedVersions below makes this a little redundant.  I think this rule was intended moreso for things like postgresql and keycloak rather than operate/optimize because those have explicit allowedVersions set.

:+1: